test/euphonic_horace_test.py

```diff
@@ -386,3 +384,59 @@ def test_optimum_chunk_mocked_psutil_virtualmemory(self, mocker):
         n_atoms = 9
         evec_bytes_per_qpt = 16*(3*n_atoms)**2
         assert quartz_coh.chunk == int(available_memory/(10*evec_bytes_per_qpt))
+
+@pytest.mark.brille
+class TestBrille:
+
+    def test_brille_euphonic_similar_results(self):
+        from euphonic.brille import BrilleInterpolator
+
+        opt_dict = {'temperature': 300}
+        fname = get_expected_output_filename('quartz', opt_dict)
+        # Ignore vals below 25 meV - hard to test
+        expected_w, expected_sf, idx_ignore = get_expected_w_sf(
+            fname, lim=25)
```

Comment:
This is more a comment about the accuracy of the Brille interpolation rather than the test or implementation, but it's a bit concerning to me that you need to set a limit of 25 meV, and moreover even then the absolute tolerance needed to pass in line 412 below is 3 meV. These numbers just seem quite large to me, implying that the Brille interpolation is not really that good. I guess it stems from using quite a coarse grid of `grid_npts=1000`?

I think the maximum frequency in quartz is around 150 meV, so the "low" frequency limit here is about 15% of the max, and the tolerance is about 2% of max... I guess it's maybe not unreasonable but I had thought Brille would be better... What if you used say, `grid_npts=5000`?

Also, what is the length of `idx_ignore` compared to `expected_w`? If it's small (say a few percent) then I guess it's ok.

Reply:
Part of it is using a coarse grid yes, but I think because quartz has a lot of degenerate frequencies perhaps brille has a hard time tracking the modes sometimes, and we end up with the odd frequency or structure factor which is a bit off, this works out ok for large numbers of q-points but makes it hard to test numerically. I've run `euphonic-brille-convergence quartz.castep_bin --brille-npts 1000 5000 10000` and here are the results:
![image](https://user-images.githubusercontent.com/18187336/217539793-5230f448-c017-4ca0-a6c5-3e2a490f61d4.png)
So overall it's pretty good, but there is the odd issue with frequencies/structure factors at certain points.

I used quartz because I wanted to have a brille-extra-only test (rather than using NaCl and having to rely on the Phonopy dependencies as well). Here is a similar run for NaCl:
![image](https://user-images.githubusercontent.com/18187336/217540455-337ed410-b1a0-4348-945c-ab2ea27ab559.png)
So the frequencies are much better but there's still a bit of discrepancy in the intensities. I could use NaCl instead but it's still just a case of fiddling with the numbers until the tests pass...

When `lim=25`, `idx_ignore` has 41 values, and `expected_w` has `8*27 = 216` values, so about 20%...

Reply:
I think it's ok to leave quartz as the test case. In fact I think the test itself is fine, it was just the thresholds which surprised me a bit. Given that it seems to be skewed by a few outliers, would it make sense to use the mean absolute frequency residual as the criteria rather than `allclose`?

Reply:
Good idea, done in [1e05b5d](https://github.com/pace-neutrons/euphonic_sqw_models/pull/11/commits/1e05b5d50359a92cd6f232e21327dabe73b93e4b).

I have also used the structure factor residuals. It is enough to fail if the proper parameters don't get passed (e.g if I don't use the correct temperature) so I think that's good enough to tell us if something goes wrong.